docs/output.md

```diff
@@ -129,6 +129,18 @@ NanoPlot is used to calculate various metrics and plots about the quality and le
 
 </details>
 
+## Digital normalization with BBnorm
+
+If the pipeline is called with the `--bbnorm` option, it will normalize sequencing depth by removing reads to 1) reduce coverage of very abundant kmers and 2) delete very rare kmers (see `--bbnorm_target` and `--bbnorm_min` parameters).
+When called in conjunction with `--coassemble_group`, BBnorm will operate on interleaved (merged) FastQ files, producing only a single output file.
+If the `--save_bbnorm_reads` parameter is set, the resulting FastQ files are saved together with log output.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bbmap/bbnorm/[sample]\*.fastq.gz
+- `bbmap/bbnorm/log/[sample].bbnorm.log
+
```

Comment:
Missing closing tags?

Reply:
You missed the closing rtags @erikrikarddaniel 😱 dev docs are broken! I'll make a PR now (skipping ci) form you to approve